python/anvill/ida.py

```diff
@@ -38,6 +38,7 @@
 from .os import *
 from .program import *
 from .type import *
+from .dwarf import *
```

Comment:
lets wrap this in a `try`...`except` block, and maybe recover, providing some kind of fake `DWARFCore` class that is a big NOP or something? I would like this to gracefully degrade, if possible.